### PR TITLE
Adding EVLR offset to las files

### PR DESCRIPTION
**Pull request by nicolas-chaulet**
It seems that the EVlr offset is never set. This PR fixes that (with a test)

### CODE REVIEW

File: test/unit/io/LasWriterTest.cpp [hunk 1]
```diff
@@ -1183,6 +1183,39 @@ TEST(LasWriterTest, pdal_add_vlr)
...
+        wo.add("filename", outfile);
+        w.setOptions(wo);
+        w.setInput(r);
+
+        PointTable t;
+        w.prepare(t);
+        w.execute(t);
+        LasTester tester;
+        LasHeader *h = tester.header(w);
+        EXPECT_GT(h->eVlrOffset(),0);
```
Comment (abellgithub) [inline test/unit/io/LasWriterTest.cpp]: There's really no reason not to test the actual value.  There are a lot of wrong values that are greater than 0.

Comment (nicolas-chaulet) [reply test/unit/io/LasWriterTest.cpp]: Sure... Will add a better test.

File: test/unit/io/LasWriterTest.cpp [hunk 2]
```diff
@@ -1189,16 +1189,19 @@ TEST(LasWriterTest, evlroffset)
 
     FileUtils::deleteFile(outfile);
     {
-        LasReader r;
+        StageFactory f;
+        Stage& r = *(f.createStage("readers.faux"));
```
Comment (abellgithub) [inline test/unit/io/LasWriterTest.cpp]: No need to read 0 points.  Just pitch this and execute the writer.

**Comment by abellgithub** [comment]
Close #2824

**Pull request merged**